i18n/zh-CN/docusaurus-plugin-content-docs/version-1.13.0/user_guide/dashboard_usage.md

```diff
@@ -124,4 +124,71 @@ HIVE流向：
 - 数据用途：选择数据使用用途
 - 数据用途说明：需申请人根据自身消费场景，简要说明使用的项目和数据的用途
 
-信息填完完成后，点击【提交】后，会将次数据消费流程正式提交待审批人审批后方可生效。
\ No newline at end of file
+信息填完完成后，点击【提交】后，会将次数据消费流程正式提交待审批人审批后方可生效。
+
+## 数据同步
+数据同步模块展示目前用户权限内同步系统所有任务列表，可以对这些任务详情查看、编辑更新和删除操作。
+点击【新建数据同步】，进入数据同步流程，需要对同步信息相关信息进行填写，可选择两种同步类型：实时或者离线。
+
+![Application detail](img/create_sync.png)
+
+### 基础信息
+
+#### 实时同步
+
+填写好相关基础信息之后即可进行下一步
```

Comment:
What basic information does it include?Please add an introduction to the key parameters

Reply:
done